src/http-driver.js

```diff
@@ -103,14 +108,12 @@ function makeHTTPDriver({eager = false} = {eager: false}) {
       .map(reqOptions => {
         let response$ = createResponse$(reqOptions)
         if (eager || reqOptions.eager) {
-          response$ = response$.replay(null, 1)
-          response$.connect()
+          response$ = response$.publishReplay(null, 1).connect()
```

Comment:
Gotcha this shouldn't work, because connect() actually returns a Subscription, not an Observable. That's why in the original code the connect() was on a different line.